setup.py

```diff
@@ -80,6 +80,7 @@
         'scripts/db_migrator.py',
         'scripts/decode-syseeprom',
         'scripts/dropcheck',
+        'scripts/disk_check.py',
```

Comment:
>disk_check [](http://example.com/codeflow?start=17&length=10)

The filename is misleading. It looks only checking, but actually checking and fixing.

Reply:
Not  fixing. A temporary/transient hold in place. If you notice, even after successful overlay, which has /etc as RW *now*, it still reports error.